docs/introduction.md

```diff
@@ -138,6 +140,43 @@ ncn-m001# sat bash
 (CONTAINER_ID) sat-container# sat status
 ```
 
+## SAT in CSM
+
+In CSM 1.3 and newer, the `sat` command is automatically available on all the Kubernetes NCNs, though it is still
+possible to install SAT as an independent product stream. Installing the SAT product stream results in additional
+supporting components being installed.
+
+- Installing the SAT product stream creates an entry for SAT in the `cray-product-catalog` Kubernetes ConfigMap.
+  Without installing the SAT product stream, there will be no entry for this version of SAT in the output of
+  `sat showrev`.
+
+- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
+  provides uninstall and activate functionality when used with the `prodmgr` command. It is still possible to uninstall
+  previous versions of the SAT product stream.
```

Comment:
Just to respond to this one quickly - the reworded version is maybe a little misleading. It's confusing but the most correct, detailed thing to say might be something like 'It is still possible to uninstall previous versions of the SAT product stream because those versions come packaged with `sat-install-utility`.

In other words, a version of `sat-install-utility` is packaged with each product stream release and is used for uninstalling that version of the product stream.

Reply:
Are the previous versions of SAT you're referring to those before SAT was available in CSM 1.3 (so SAT 2.3 and older)? Is this a distinction we should make if SAT 2.4 and newer can't be uninstalled unless the full SAT product stream is installed on an EX system? Maybe we add something like the following as a new line indented underneath this bullet:

NOTE: In SAT 2.3 and older, SAT was only available to install as a separate product stream. Because these versions were packaged with `sat-install-utility`, it is still possible to uninstall these versions of SAT.

Reply:
Yeah, that is pretty much right.

Although to add to the confusion, 2.4 is really a version of the product stream, so it doesn't exactly make sense to say "SAT 2.4 is in CSM 1.3", you might say something like "the version of the SAT CLI that is in SAT 2.4 is also in CSM 1.3" or something.

I like the note there, I think that's totally accurate to say. Maybe we could condense it all and say,
```
- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
  provides uninstall and activate functionality when used with the `prodmgr` command. In SAT 2.3 and older, SAT was
  only available to install as a separate product stream. Because these versions were packaged with
  `sat-install-utility`, it is still possible to uninstall these versions of SAT.
```

I'd be open to some more feedback here, if you want to further refine that.

Reply:
I'm ok with including the note with the bullet if we put parenthesis around it. I'd like to somehow distinguish visually that this is additional information for older SAT versions. That way it doesn't disrupt too much the flow of the bulleted list overall. What do you think of the following?

- The `sat-install-utility` container image is only available with the full SAT product stream. This container image
  provides uninstall and activate functionality when used with the `prodmgr` command. (In SAT 2.3 and older, SAT was
  only available to install as a separate product stream. Because these versions were packaged with
  `sat-install-utility`, it is still possible to uninstall these versions of SAT.)

Reply:
Yeah, that looks awesome, thanks!

Reply:
I implemented the suggestion in 45fae7c23fd959bdca55482f63824e0422d6509c